src/App.tsx

```diff
@@ -1,7 +1,47 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type Props = {};
+
+type State = {
+  key: string;
+};
+
+export class App extends React.Component<Props, State> {
```

Comment:
you don't have to put `type Props` in this case because there is no type definition (If I am not mistaken), you just can put `<{}, State>` on line 9.